clients/golang/go.mod

```diff
@@ -0,0 +1,5 @@
+module taskforce.sh/bullmq_proxy_client
```

Comment:
If it is going to be served with GitHub, this should be changed to "github.com/taskforcesh/bullmq-proxy/clients/golang" here, and similarly, all imports should be changed accordingly.

Reply:
I see. It would be nicer not to need to append the path to the module as this is a monorepo with packages for several platforms, I guess that would require a different solution, like serving it from a different host right?

Reply:
Yeah, I prefer the mono repo approach as it will be easier to keep the go library and the proxy in sync.

Reply:
or if you want a different host, it's called vanity url, but you have to serve the vanity url server from another machine, be aware that I'm not talking about hosting the package, the package will still be available on any git provider

Reply:
instead of hosting this server directly you can do a simple implementation yourself any serverless functions platform can handle it

Reply:
I added what you need directly in the first comment

Reply:
For monorepos: https://stackoverflow.com/questions/72356519/how-to-set-go-vanity-import-url-to-repository-subdirectory-in-a-monorepo

Reply:
I understand. I rather not put the extra burden of maintaining an extra host, even if it is a lambda... these things breaks sometimes and could affect users when they need it the most. So I guess I will go the github route for now.

Reply:
Thanks for all the feedback :)

Reply:
You're welcome, I will contribute for more clients soon!